### PR TITLE
Update to 0.5.10, conda-smithy 2.2.2, and use sha256 as verification hash

### PR DESCRIPTION
**Pull request by lannybroo**

### CODE REVIEW

**Comment by frol** [comment]
@lannybroo Thank you!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ build:
 requirements:
   build:
     - python
+    - setuptools
```
Comment (jakirkham) [inline recipe/meta.yaml]: Needs these [arguments]( https://github.com/conda-forge/staged-recipes/blob/1caddd635ce1a2fbe477bc216878c8ea482e7cf1/recipes/example/meta.yaml#L26 ) when using `setuptools`. Can fix in a build number bump.

Comment (frol) [inline recipe/meta.yaml]: @jakirkham Good catch! Fixed.

Comment (jakirkham) [inline recipe/meta.yaml]: Thanks for fixing this, @frol.

**Pull request merged**